packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -0,0 +1,178 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+
+import '../../all-components';
+import type { WaferMapDie, WaferMapColorsScale } from '../types';
+import {
+    WaferMapQuadrant,
+    WaferMapOrientation,
+    WaferMapColorsScaleMode
+} from '../types';
+import {
+    HIGHLIGHTEDVALUESETS,
+    WAFERMAPDIESETS,
+    WAFERMAPCOLORSCALESETS
+} from './sets';
+
+interface WaferMapArgs {
+    quadrant: WaferMapQuadrant;
+    orientation: WaferMapOrientation;
+    maxCharacters: number;
+    dieLabelsHidden: boolean;
+    dieLabelsSuffix: string;
+    colorsScaleMode: WaferMapColorsScaleMode;
+    highlightedValues: number[];
+    dies: WaferMapDie[];
+    colorScale: WaferMapColorsScale;
+}
+
+const metadata: Meta<WaferMapArgs> = {
+    title: 'WaferMap',
+    parameters: {
+        docs: {
+            description: {
+                component: 'A WaferMap description'
+            }
+        },
+        actions: {
+            handles: ['click', 'mouseover']
+        }
+    },
+    render: createUserSelectedThemeStory(html`
+        <nimble-wafer-map
+            id="waferMapEl"
+            quadrant="${x => x.quadrant}"
+            orientation="${x => x.orientation}"
+            maxCharacters="${x => x.maxCharacters}"
+            dieLabelsHidden="${x => x.dieLabelsHidden}"
+            dieLabelsSuffix="${x => x.dieLabelsSuffix}"
+            colorsScaleMode="${x => x.colorsScaleMode}"
+            :highlightedValues="${x => x.highlightedValues}"
+            :colorScale="${x => x.colorScale}"
+            :dies=${x => x.dies}
+        >
+        </nimble-wafer-map>
+    `),
+    args: {
```

Comment:
Unless there's a strong reason to order the args like you have, could we put them in alphabetical order?

Reply:
applied